meta-lmp-base/recipes-containers/docker/docker-ce_%.bbappend

```diff
@@ -3,9 +3,19 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 SRC_URI_append = " \
```

Comment:
Can we add a comment above here with the tracking location of the docker patch for max download retries?

Reply:
done